src/components/Table/components/Row.tsx

```diff
@@ -334,4 +334,6 @@ export type GridDataRow<R extends Kinded> = {
   initSelected?: boolean;
   /** Whether row can be selected */
   selectable?: false;
+  /** Whether this row should infer its selected state based on its children's selected state */
+  inferSelectedState?: false;
```

Comment:
Thought: I was kind of wondering if this should be a table-level setting instead of setting it onsey/twosy on grid rows, but I suppose it is more flexible, like you could have some parent rows with it false, some with it true... :shrug: